.github/workflows/ci.yml

```diff
@@ -24,10 +24,10 @@ jobs:
                   git config user.name github-actions
                   git config user.email github-actions@github.com
 
-            - name: Use Node.js 14.x
+            - name: Use Node.js 18.x
               uses: actions/setup-node@v3
               with:
-                  node-version: 14
+                  node-version: 18
```

Comment:
we use in newer projects already 20, and 20 is the current LTS. Maybe you could switch here too?

Reply:
Would probably be okay, as long as it still works with Node 18